tasks/section_3_Network_Configuration.yaml

```diff
@@ -19,6 +19,11 @@
       when: not disable_IPv6_in_grub
     - name: 3.1.1 Disable IPv6 - update grub
       shell: update-grub
+    - name: 3.1.1 Disable IPv6 - Disable ipv6 ufw policies if ipv6 is not enabled 
+      replace:
+        path: /etc/default/ufw
+        regexp: '^(IPV6=yes)'
+        replace: 'IPV6=no'
```

Comment:
What happens when there exist already ipv6 rules in ufw prior to changing this configuration?

Reply:
The change will disable any previously existing IPv6 rules in UFW. According to the "when" statement in `when: not IPv6_is_enabled`, this change is going to be executed only if ipv6 is not needed, so it won't matter if there were previous ipv6 rules.

Reply:
Fair enough.